src/npctalk.cpp

```diff
@@ -1980,11 +1982,17 @@ void talk_effect_fun_t::set_bulk_trade_accept( bool is_trade, bool is_npc )
                     }
                     for( int i = 0; i < buyer_has; i++ ) {
                         seller->i_add( pay );
-                        price -= d.beta->value( pay );
+                        price -= value;
                     }
+                } else {
+                    popup( _( "%1$s can only pay you in worthless!  Report this is a bug!" ) );
```

Comment:
```suggestion
                    popup( _( "%1$s can only pay you in worthless!  Report this as a bug!" ) );
```
Also, maybe change popup to `debugmsg`? The same for several lines below.

Reply:
I kind of hate debugmsg but you're probably right.

Reply:
Reworded and now using debugmsg.